frontend/vuejs/src/views/Tables/listCourse.vue

```diff
@@ -47,64 +62,109 @@
         </div>
         <div>
             <b-table striped hover :items="data" :fields="fields">
+                <template v-slot:cell(is_active)="row">
+                    <p v-if="row.item.is_active == 1"> {{ $t("course_screen.course.active") }}</p>
+                    <p v-else> {{ $t("course_screen.course.inactive") }}</p>
+                </template>
 
-                <template slot="actions" slot-scope="row">
-                    <b-button class="btn btn-danger" @click="onDeleteTask(row.item.id)">delete</b-button>
-                    <b-button class="btn btn-success" v-bind:to="'editCourse/' + row.item.id">update
+                <template v-slot:cell(actions)="row">
+                    <b-button class="btn btn-danger" @click="deleteCourse(row.item.id)">
+                        {{ $t("course_screen.course.delete") }}
+                    </b-button>
+                    <b-button class="btn btn-success" v-bind:to="'editCourse/' + row.item.id">
```

Comment:
1. sử dụng shorthand thay cho v-bind:to
2. sử dụng router name thay cho router path